scripts/verify

```diff
@@ -35,6 +35,10 @@ case $i in
     istio="${i#*=}"
     shift
     ;;
+  --role=*)
```

Comment:
Not clear what this is for. `--additional_clusterrole`?

Reply:
What about `--additional_deployer_role`